processing/src/main/java/org/apache/druid/query/DruidMetrics.java

```diff
@@ -52,6 +52,8 @@
 
   public static final String PARTITION = "partition";
 
+  public static final String INGEST_METADATA = "ingestMetadata";
```

Comment:
This is the key used in the metrics that are emitted. Thoughts on naming this key?